src/convertToObject.js

```diff
@@ -7,6 +7,16 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stringArr = sourceString
+    .split(';')
+    .map((el) => el.split(':'))
+    .filter((el) => el.length > 1);
```

Comment:
While you are correctly chaining `split`, `map`, and `filter`, the indentation of chained methods is not consistent. According to the code style guidelines, each method should start on a new line with the same indentation to improve readability.